From: Jo-Philipp Wich <jo@mein.io>
Date: Thu, 26 Jan 2017 15:30:00 +0100
Subject: [PATCH] config: adjust default values

Adjust the default interval settings to be better suited for slower embedded
devices.

Signed-off-by: Jo-Philipp Wich <jo@mein.io>

--- a/cfg/vnstat.conf
+++ b/cfg/vnstat.conf
@@ -98,13 +98,13 @@ DaemonGroup ""
 TimeSyncWait 5
 
 # how often (in seconds) interface data is updated
-UpdateInterval 30
+UpdateInterval 60
 
 # how often (in seconds) interface status changes are checked
-PollInterval 5
+PollInterval 30
 
 # how often (in minutes) data is saved to file
-SaveInterval 5
+SaveInterval 30
 
 # how often (in minutes) data is saved when all interface are offline
 OfflineSaveInterval 30
